packages/contracts/deploy/new/40_finalize-managing-dao/00_grant-permissions.ts

```diff
@@ -4,6 +4,7 @@ import {DeployFunction} from 'hardhat-deploy/types';
 import {Operation} from '../../../utils/types';
 import {getContractAddress, managePermissions, Permission} from '../../helpers';
 import {PluginRepo__factory} from '../../../typechain';
+import {DAO__factory} from '../../../typechain/osx-versions/v1_2_0/contracts/core/dao/DAO.sol';
```

Comment:
Why 1.2.0? Shouldn't it always be the current version?

Otherwise we'll always need manual intervention here

Reply:
🙏 This came from auto-completion. Fixed in 1ac77f580109dd5c4612e7a70899fae68135606f